src/hardware/serialport/serialport.cpp

```diff
@@ -1002,6 +1003,21 @@ void CSerial::setCTS (bool value) {
 	//else no change
 }
 
+constexpr std::tuple<uint8_t, uint8_t> baud_to_regs(uint32_t baud_rate)
+{
+	// Cap the lower-bound to 300 baud. Although the first 1950s modem
+	// offered 110 baud, by the time DOS was available 8-bit ISA modems
+	// offered at least 300 and even 1200 baud.
+	baud_rate = std::max(300u, baud_rate);
+
+	const auto max_baud = static_cast<uint16_t>(115200u);
```

Comment:
I think `constexpr uint16_t max_baud = 115200;` would be ok and would not introduce any warnings about converting 32-bit value to 16-bit (because we're initializing variable based on literal). But it's a nitpick.

Reply:
Thanks for catching this; fixed.